releng/release-tool/release-tool.go

```diff
@@ -386,17 +395,20 @@ func (r *releaseData) forkProwJobs() error {
 	gitbranch := fmt.Sprintf("%s_%s_%s_configs", r.org, r.repo, r.newBranch)
 
 	_, err := gitCommand("-C", r.infraDir, "checkout", "-b", gitbranch)
-	// if err, then branch probably already exists
+
+	// if err, then branch likely already exists
```

Comment:
Could this potentially lead to a situation where we take the branch from a not-anymore correct commit? Would it make sense to use `-B` instead. Basically create-if-necessary and set to the current commit?

Reply:
yep, fixed